src/commands/lambda/instrument.ts

```diff
@@ -343,3 +415,4 @@ InstrumentCommand.addOption('service', Command.String('--service'))
 InstrumentCommand.addOption('environment', Command.String('--env'))
 InstrumentCommand.addOption('version', Command.String('--version'))
 InstrumentCommand.addOption('extraTags', Command.String('--extra-tags'))
+InstrumentCommand.addOption('sourceCodeIntegration', Command.Boolean('-sci,--source-code-integration'))
```

Comment:
I think the short version of the flag (`-sci`) should be one letter long (`-s`). Usually these short flags are only one letter long, and can be combined, so `-sci` could also be `-s -c -i`. See https://nullprogram.com/blog/2020/08/01/

Reply:
Agreed, the previous PR had `-sci` so I didn't know whether or not this discussion was already had. 👍